### PR TITLE
Implement newsletter signup

### PR DESCRIPTION
**Pull request by duerdod**
I think using
  1. double quotes on html 
  2. having resource routes prefixed with '_' 

makes sense

### CODE REVIEW

**Comment by ankarhem** [comment]
We can also make sure we're retargeting the input on errors with a ref + useEffect.

**Comment by duerdod** [comment]
> 

I did implement this + resetting the form on success to begin with, but decided to remove it since it felt cluttered. I may have another go on it.

**Review by ankarhem — APPROVED** [review]

File: app/components/Footer/index.tsx [hunk 1]
```diff
@@ -1,13 +1,94 @@
...
+              className="flex-shrink-0 px-4 py-2 text-base font-semibold text-white bg-gray-500 rounded shadow-md hover:bg-gray-600 focus:outline-none focus:ring-1 focus:hover:ring focus:ring-gray-600 aria-disabled:bg-gray-100/95 aria-disabled:cursor-not-allowed"
+              type="submit"
+            >
+              Subscribe
+            </button>
+            <div className="relative pt-1 min-w-[100%] w-100">
+              <p
+                aria-hidden={data?.subscribed ? false : true}
+                className="text-sm absolute aria-hidden:invisible"
+                tabIndex={state === 'success' ? 0 : -1}
```
[the remark this answers is not in the document]
Comment (duerdod) [reply app/components/Footer/index.tsx:65]: Do we? If it has tabindex 0 on success, it means the next tab will be the success message, meaning it'll read it? But also, I see that I'm already focusing on it on success in the effect. Maybe that's enough? 

I'll change both messages to have tabindex -1.

[the remark this answers is not in the document]
Comment (duerdod) [reply app/components/Footer/index.tsx:65]: But also changing the tabindex with javascript can be kinda annoying right?

**Pull request merged**